lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -48,7 +75,7 @@
     <button class="action action-cancel">
         <%=
             interpolate_text(
-                _.escape(gettext("Cancel {span_start}a new team{span_end}")),
+                _.escape(gettext("Cancel {span_start} team {span_end}")),
```

Comment:
great @marcotuts.  If that condition applies here, then lets not cancel the cancel button ;)  I don't have any background on this control, I was just reviewing code.  If we are going to keep the cancel button, I prefer to add some contextual screen reader text that clarifies exactly what it is that is being cancelled as @catong originally suggested.